src/plugin/memory-webview-main.ts

```diff
@@ -125,7 +128,7 @@ export class MemoryWebview implements vscode.CustomReadonlyEditorProvider {
         };
 
         if (!panel) {
-            panel = vscode.window.createWebviewPanel(MemoryWebview.ViewType, 'Memory Inspector', vscode.ViewColumn.Active, options);
+            panel = vscode.window.createWebviewPanel(MemoryWebview.ViewType, `Memory ${this.panelIndices++}`, vscode.ViewColumn.Active, options);
```

Comment:
Theoretically we can catch when webview panel is closed and reuse that number. It will however be probably overcomplicated so it is much better like this 👍

Reply:
Right, that was also my thinking. It would add a bit of extra code without too much value. But I'm certainly open to it if you prefer.